domain/model/state/controllerstate.go

```diff
@@ -996,16 +1032,19 @@ AND       u.name = $dbUserName.name
 	name := dbUserName{Name: userName.Name()}
 	modelStmt, err := s.Prepare(q, name, dbModelSummary{})
 	if err != nil {
-		return nil, errors.Annotatef(err, "preparing get model summary for user statement")
+		return nil, errors.Capture(err)
+	}
+
+	type controllerUUID struct {
```

Comment:
Should this struct be in types.go
In fact, we don't need it, we have a `dbUUID` struct we can use.